pkg/kafka/configkafka/config.go

```diff
@@ -53,6 +53,13 @@ type ClientConfig struct {
 	// replica selection when supported by the brokers. This maps to Kafka's
 	// standard "client.rack" setting. By default, this is empty.
 	RackID string `mapstructure:"rack_id"`
+
+	// UseLeaderEpoch forces the collector to clear the leader epoch from
```

Comment:
Sorry if I was unclear, when I said to invert the name I also meant to invert the meaning. So "use_leader_epoch: true" is what franz-go would do be default -- it would use the leader epoch returned by the brokers. Setting to false would disable KIP-320. Does this sound sensible?